.github/workflows/analysis_workflow.yml

```diff
@@ -57,12 +67,13 @@ jobs:
                 -DOPENEXR_RUN_FUZZ_TESTS='OFF' \
                 -DOPENEXR_ENABLE_THREADING='ON'
         working-directory: _build
-      - name: Build
+      - name: Build OpenEXR with build-wrapper
         shell: bash
         run: |
-          cmake --build . \
-                --target install \
-                --config Release
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${PWD}/Half:${PWD}/Imath
```

Comment:
The /Half part is obsolete, there is no more Half library. And with a static build there should be no need for /Imath either, or LD_LIBRARY_PATH at all, which makes things simpler.

Reply:
Thanks I'll update this here and think I need to update it on the Imath analsysis workflow as well.